feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableItemDetailContent.kt

```diff
@@ -68,14 +75,25 @@ fun TimetableItemDetailContent(
 }
 
 @Composable
-private fun DescriptionSection(description: String, modifier: Modifier = Modifier) {
+private fun DescriptionSection(
+    description: String,
+    modifier: Modifier = Modifier,
+) {
+    var isExpanded by rememberSaveable { mutableStateOf(false) }
+
     Column(modifier = modifier) {
         Text(
             text = description,
             fontSize = 16.sp,
             color = MaterialTheme.colorScheme.onSurface,
+            maxLines = if (isExpanded) Int.MAX_VALUE else 5,
+            overflow = TextOverflow.Ellipsis,
             modifier = Modifier.padding(start = 16.dp, end = 16.dp),
         )
+        ReadMoreOutlinedButton(
```

Comment:
Your codes are great!
But I think we don't have to show ReadMoreOutlinedButton when expanding so can you surround it with if?

Reply:
I can understand your concerns. But I think nobody will care about this. Because we can reopen the screen.

Reply:
[0a855e0](https://github.com/DroidKaigi/conference-app-2023/pull/582/commits/0a855e0f9e8e61e6559b5e131eefe6075d1d82ca)
Thanks for your kind reply.